src/test/lib/operators/join_hash/join_hash_steps_test.cpp

```diff
@@ -98,7 +98,7 @@ TEST_F(JoinHashStepsTest, LargeHashTableExistenceOnly) {
 
 TEST_F(JoinHashStepsTest, MaterializeAndBuildWithKeepNulls) {
   const size_t radix_bit_count = 0;
-  std::vector<std::vector<size_t>> histograms;
+  pmr_vector<pmr_vector<size_t>> histograms;
```

Comment:
Can you briefly elaborate on why you use the pmr_vector here?

Reply:
Because `materialize_input` expects a ´pmr_vector` of `pmr_vector`s.